### PR TITLE
LPD-42407 User should be notifiable after the workflow task in completed, so that they can still click on the notification

### PR DESCRIPTION
**Pull request by devpedrotavares**
Related ticket: [LPD-42407](https://liferay.atlassian.net/browse/LPD-42407)

[LPD-42407]: https://liferay.atlassian.net/browse/LPD-42407?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: modules/apps/portal-workflow/portal-workflow-kaleo-test/src/testIntegration/java/com/liferay/portal/workflow/kaleo/internal/runtime/integration/test/WorkflowTaskManagerImplTest.java [hunk 1]
```diff
@@ -990,6 +990,25 @@ public void testIsNotifiableUser() throws Exception {
 		Assert.assertTrue(
 			_workflowTaskManager.isNotifiableUser(
 				user.getUserId(), workflowTask.getWorkflowTaskId()));
+
+		PermissionChecker userPermissionChecker =
+			PermissionCheckerFactoryUtil.create(user);
+
+		PermissionThreadLocal.setPermissionChecker(userPermissionChecker);
```
Comment (linolaoj) [inline modules/apps/portal-workflow/portal-workflow-kaleo-test/src/testIntegration/java/com/liferay/portal/workflow/kaleo/internal/runtime/integration/test/WorkflowTaskManagerImplTest.java:997]: Hi @devpedrotavares, should we get the original value of the permission checker to restore it on the PermissionThreadLocal after we finish using it?

**Review by linolaoj — CHANGES_REQUESTED** [review]

File: modules/apps/portal-workflow/portal-workflow-kaleo-test/src/testIntegration/java/com/liferay/portal/workflow/kaleo/internal/runtime/integration/test/WorkflowTaskManagerImplTest.java (hunk 1 above)
Comment (devpedrotavares) [reply modules/apps/portal-workflow/portal-workflow-kaleo-test/src/testIntegration/java/com/liferay/portal/workflow/kaleo/internal/runtime/integration/test/WorkflowTaskManagerImplTest.java]: Hi @linolaoj , Yes, I just changed it!

**Comment by devpedrotavares** [comment]
ci:test:sf

**Comment by devpedrotavares** [comment]
ci:test:relevant

**Comment by devpedrotavares** [comment]
ci:test:workflow

**Comment by devpedrotavares** [comment]
ci:report:72201946

**Pull request closed without merging**